data/human/boarding missions.txt

```diff
@@ -703,3 +703,238 @@ mission "Turn in Tarquin"
 		conversation
 			"You haul captain Tarquin out of your brig and bring him to the portside police station, presenting evidence from your suit cameras and notarized witness statements compiled from your crew members attesting to his crimes of being involved in slavery. The officer on duty thanks you for the service you've rendered, and takes Tarquin into custody. He shoots one final black look at you that sends a shiver up your spine."
 				decline
+
+mission "Boarding Derelict"
+	boarding
+	repeat
+	invisible
+	deadline 3
+	source
+		government "Derelict (Boardable)"
+	destination "Earth"
```

Comment:
We should have
```
to complete
	never
```
so that landing on Earth doesn't finish this mission while you're being chased and the deadline isn't up.